syncopy/io/load_nwb.py

```diff
@@ -197,7 +197,7 @@ def load_nwb(filename, memuse=3000, container=None):
 
         evtData = EventData(dimord=["sample","eventid","chans"], filename=filename)
         h5evt = h5py.File(evtData.filename, mode="w")
-        evtDset = h5evt.create_dataset("data", dtype=np.result_type(*ttlDtypes),
+        evtDset = h5evt.create_dataset("data", dtype=int,
```

Comment:
Okay that looks much simpler of course.. we can be sure only integers are to be expected here?

Reply:
No they are floats but they need to be converted otherwise it crashes.

Reply:
Ok.. so does that mean there could be events encoded with floats? I think we only enforce integere types for `SpikeData` now, not for `EventData`?!

Reply:
Events should always be integers but they're not stored like that in NWB. No it's enforced for both, this crashes now.

Reply:
Uh, but apparently rightfully so :slightly_smiling_face: Ok then let's do it like you proposed!

Reply:
We could print a warning that we're casting?

Reply:
Maybe at the debug level, we have proper logging now with #409 But as you said, the nwb reader should anyways be considered experimental :sweat_smile: